data.sql

```diff
@@ -1 +1,4 @@
-INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Argumon',TO_DATE('2020-02-03', 'YYYY-MM-DD'),0,true,10.23), (2,'Gabumon',TO_DATE('2018-11-15', 'YYYY-MM-DD'),2,true,8), (3,'Pikachu',TO_DATE('2021-01-07', 'YYYY-MM-DD'),1,false,15.04), (4,'Devimon',TO_DATE('2017-05-12', 'YYYY-MM-DD'),5,true,11);
\ No newline at end of file
+
+
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1,'Argumon',TO_DATE('2020-02-03', 'YYYY-MM-DD'),0,true,10.23), (2,'Gabumon',TO_DATE('2018-11-15', 'YYYY-MM-DD'),2,true,8), (3,'Pikachu',TO_DATE('2021-01-07', 'YYYY-MM-DD'),1,false,15.04), (4,'Devimon',TO_DATE('2017-05-12', 'YYYY-MM-DD'),5,true,11), (5,'Charmander',TO_DATE('2020-02-08', 'YYYY-MM-DD'),0,false,11), (6,'Plantmon',TO_DATE('2021-11-15', 'YYYY-MM-DD'),2,true,-5.7), (7,'Squirtle',TO_DATE('1993-04-02', 'YYYY-MM-DD'),3,false,-12.13), (8,'Angemon',TO_DATE('2005-06-12', 'YYYY-MM-DD'),1,true,-45), (9,'Boarmon',TO_DATE('2005-06-07', 'YYYY-MM-DD'),7,true,20.4), (10,'Blossom',TO_DATE('1998-10-13', 'YYYY-MM-DD'),3,true,17), (11,'Ditto',TO_DATE('2022-05-14', 'YYYY-MM-DD'),4,true,22);
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :100: :100: 
- [ ] The `weight_kg` for `Charmander` is meant to be in the negative i.e. `-11kg`. Currently what you have is `11kg`. I'll suggest you update this and rerun the queries because this could affect the outcome of your previous queries. :+1: :smile: 
- [ ] Also I noticed that in your `data.sql` file, you merged the data for this milestone with the one of the previous milestones. I'll suggest that you actually separate them as it will help to make it clear. the current data that was inserted and also the milestone you are currently working on. :+1: :+1:

Reply:
I think the value for charmandar is supposed to be positive as it is because one of the action points read as follow
-- Update all animals' weights that are negative to be their weight multiplied by -1
UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
This will make all negative data to be positive. Kindly check...

I have worked on separating the new values in data.sql as uggested.